src/assets/css/_sass/components/_projects.scss

```diff
@@ -0,0 +1,38 @@
+.projects__hero {
+  color: $color-grey--lightest;
+  padding-top: 100px; // need to pad every container to account for the header ??? How should we do this?
```

Comment:
We should give every container a padding-top equal to the height of the header; currently `100px`

Reply:
Not a fan of the static height. I'll look into a better solution. Can you tag this with a #todo